dhis-2/dhis-web-api/src/main/resources/openapi/TrackedEntitiesExportController.md

```diff
@@ -107,54 +107,73 @@ entities. Only tracked entities with an enrollment into the `program` will be re
 
 ### `*.parameter.TrackedEntityRequestParams.enrollmentStatus`
 
-Get tracked entities with an enrollment in the given status.
+Get tracked entities that have at least one enrollment with the status specified.
+
+Valid options are:
+- `ACTIVE`
```

Comment:
I don't think we need to mention the options here 🤔 please generate the spec and check. I think our spec picks up the enum and shows the valid options already. You would only list them to add some details like you do with modes.

Reply:
Yes you are right. I removed it. The spec shows "Allowed Values. "